pom.xml

```diff
@@ -136,6 +136,38 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+	        <dependency>
+	            <groupId>net.sourceforge.javacsv</groupId>
+	            <artifactId>javacsv</artifactId>
+	            <version>2.0</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>commons-io</groupId>
+	            <artifactId>commons-io</artifactId>
+	            <version>2.4</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>org.mapdb</groupId>
+	            <artifactId>mapdb</artifactId>
+	            <version>1.0.8</version>
+	        </dependency>
+	        <dependency>
+	            <groupId>io.leonard</groupId>
+	            <artifactId>gtfs-realtime-bindings</artifactId>
+	            <version>0.0.7</version>
+	            <exclusions>
```

Comment:
The 'official' repository now seems to be: https://github.com/MobilityData/gtfs-realtime-bindings

```
This project was originally created by Google - MobilityData started to maintain the project in early 2019.
```

I don't know what io.leonard is, but they recommend these coordinates here: https://github.com/MobilityData/gtfs-realtime-bindings/blob/master/java/README.md#add-the-dependency

```xml
<dependency>
  <groupId>io.mobilitydata.transit</groupId>
  <artifactId>gtfs-realtime-bindings</artifactId>
  <version>0.0.7</version>
</dependency>
```

Reply:
https://search.maven.org/search?q=a:gtfs-realtime-bindings

`io.mobilitydata.transit` does not provide a 0.0.7 release

Reply:
But it seems to be here: https://mvnrepository.com/artifact/io.mobilitydata.transit/gtfs-realtime-bindings/0.0.7 ?

Reply:
Hm, but it does not work. See also this comment: https://github.com/MobilityData/gtfs-realtime-bindings/pull/70#issuecomment-726234803

Reply:
I've switched to 0.0.5. We can still update the lib in a later PR

Reply:
> But it seems to be here: https://mvnrepository.com/artifact/io.mobilitydata.transit/gtfs-realtime-bindings/0.0.7 ?

> Note: this artifact is located at JCenter repository (https://jcenter.bintray.com/)

Reply:
Ok sure, 0.0.5 seems to use protobuf 3.6.1